ReportUtilityBills/measure.rb

```diff
@@ -120,9 +120,20 @@ def energyPlusOutputRequests(runner, user_arguments)
 
     fuels = setup_fuel_outputs()
 
+    hpxml_fuel_map = { FT::Elec => HPXML::FuelTypeElectricity,
+                       FT::Gas => HPXML::FuelTypeNaturalGas,
+                       FT::Oil => HPXML::FuelTypeOil,
+                       FT::Propane => HPXML::FuelTypePropane,
+                       FT::WoodCord => HPXML::FuelTypeWoodCord,
+                       FT::WoodPellets => HPXML::FuelTypeWoodPellets,
+                       FT::Coal => HPXML::FuelTypeCoal }
+
     # Fuel outputs
-    fuels.each do |(fuel_type, _is_production), fuel|
+    fuels.each do |(fuel_type, is_production), fuel|
       fuel.meters.each do |meter|
+        next unless @hpxml.has_fuel(hpxml_fuel_map[fuel_type])
```

Comment:
Here we don't request output meters for fuels that the HPXML file does not have. 👍